README.md

```diff
@@ -3,6 +3,9 @@ A place to organize style guides, best practices, tools, and techniques for Stan
 
  - [Best Practices](/best-practices)
    - [Documentation](/best-practices/documentation)
+   - [How To - Readme](/best-practices/howto_readme.md)
+   - [Licensing](/best-practices/licensing.md)
+   - [Version Control](/best-practices/version_control.md)
```

Comment:
Remove lines 5-8 and leave only the link to the best-practices README.  The reason is to avoid extra management of the top level index for changes to the lower level README.  (Less maintenance is better.)